microlith/loki/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -76,6 +76,22 @@ groups:
           description: Node {{ $labels.hostname }} has reported errors due to long connection names (known issue MB-34280).
           remediation: Reduce the lengths of your nodes' hostnames, or contact Couchbase Technical Support.
 
+      - alert: CB90042-segmentationFaults
+        expr: sum( (count_over_time({file="diag.log"} |= "segfault" [1h]) + (count_over_time({file="couchbase.log"} |= "segfault" [1h]) )>0
```

Comment:
Neither `diag.log` nor `couchbase.log` exist on the server (they're generated by cbcollect_info), so we can't use them here. Here's the log files present on my CBS (Docker):

```
root@b85522bff897:/# ls /opt/couchbase/var/lib/couchbase/logs
analytics_access.log                analytics_error.log   b85522bff897-2022-01-12T11-11-08-audit.log  fts.log                   indexer_stats.log         metakv.log      rebalance
analytics_cbas_debug.log            analytics_info.log    babysitter.log                              goxdcr.log                info.log                  ns_couchdb.log  reports.log
analytics_dcp_failed_ingestion.log  analytics_trace.json  couchdb.log                                 http_access.log           json_rpc.log              projector.log   stats.log
analytics_dcpdebug.log              analytics_warn.log    debug.log                                   http_access_internal.log  mapreduce_errors.log      prometheus.log  views.log
analytics_debug.log                 audit.log             error.log                                   indexer.log               memcached.log.000000.txt  query.log       xdcr_target.log
```

Reply:
ah thats an issue, do you know where segfaults would be stored otherwise then?